CSharp/BatchExplorer/Views/MainView.xaml

```diff
@@ -1094,6 +1095,40 @@
                                 <toolkit:BusyIndicator IsBusy="{Binding LowerRightSpinnerIsVisible}"/>
                             </Grid>
                         </TabItem>
+                        <TabItem Header="Linked Storage Files">
+                            <Grid>
+                                <DataGrid ItemsSource="{Binding Path=SelectedTask.LinkedStorageOutputFiles}"
+                                          SelectedItem="{Binding Path=SelectedTask.SelectedTaskLinkedStorageFile}"
+                                          Visibility="{Binding Path=SelectedTask.LinkedStoragHasOutputFiles, Converter={StaticResource BVC}, FallbackValue=Hidden}">
+                                    <DataGrid.InputBindings>
+                                        <MouseBinding
+                                            MouseAction="LeftDoubleClick"
+                                            Command="{Binding OpenTaskLinkedStorageFile}"
+                                            CommandParameter="{Binding SelectedTask.SelectedTaskLinkedStorageFile/}"/>
+                                    </DataGrid.InputBindings>
+                                    <DataGrid.ContextMenu>
+                                        <ContextMenu>
+                                            <MenuItem Header="Open File"
+                                                Command="{Binding OpenTaskLinkedStorageFile}"/>
+                                            <MenuItem Header="Download File"
+                                                Command="{Binding DownloadTaskLinkedStorageFile}"/>
```

Comment:
Yes - I duplicated what the existing code does. Even download UI downloads the file and then opens it. Again, I am game for changing the behavior for example to handle pics better, but I think it's outside the scope of my changes, and can be done in another commit